### PR TITLE
ZCS-8089 : To make Zimbrax run on 8.8 backened with  zimbraMailURL set to other than '/'

### PR DESCRIPTION
**Pull request by auminpatel**

### CODE REVIEW

**Review by silentsakky — APPROVED** [review]

File: conf/jetty/jetty.xml.production [hunk 1]
```diff
@@ -606,20 +606,11 @@
...
-		<New class="org.eclipse.jetty.rewrite.handler.RewritePatternRule">
-		    <Set name="pattern">/*</Set>
-		    <Set name="replacement">%%zimbraMailURL%%</Set>
-		</New>
-	    </Arg>
-        </Call>      
-
-        <Call name="addRule">
+         
+		    <Call name="addRule">
```
Comment (silentsakky) [inline conf/jetty/jetty.xml.production]: indentation issue

File: conf/jetty/jetty.xml.production [hunk 2]
```diff
@@ -639,17 +630,17 @@
...
             </New>
           </Arg>
         </Call>
-        
+
         <Call name="addRule">
           <Arg>
             <New class="org.eclipse.jetty.rewrite.handler.HeaderPatternRule">
-              <Set name="pattern">zimbrax/.*/assets/*</Set>
+              <Set name="pattern">/zimbrax/.*/assets/*</Set>
```
Comment (silentsakky) [inline conf/jetty/jetty.xml.production:643]: don't we need %%zimbraMailURL%% here?

**Review by BhavinBathani — APPROVED** [review]

File: conf/jetty/jetty.xml.production (hunk 2 above)
Comment (auminpatel) [reply conf/jetty/jetty.xml.production:643]: pattern having /zimbrax in the url let it be /zimbra/zimbrax . or only /zimbra all pattern are obeyed  by above rule so caching rules are getting applied

File: conf/jetty/jetty.xml.production (hunk 1 above)
Comment (auminpatel) [reply conf/jetty/jetty.xml.production]: Done

**Review by silentsakky — APPROVED** [review]

**Pull request merged**